types/src/front/assistant/actions/websearch.ts

```diff
@@ -10,15 +12,27 @@ export type WebsearchConfigurationType = {
   forceUseAtIteration: number | null;
 };
 
-export type WebsearchActionOutputType = {
-  results: WebsearchResultType[];
-};
+export const WebsearchResultSchema = t.type({
+  title: t.string,
+  snippet: t.string,
+  link: t.string,
+});
 
-export type WebsearchResultType = {
-  title: string;
-  snippet: string;
-  link: string;
-};
+export const WebsearchActionOutputSchema = t.union([
```

Comment:
NIT: It does not work to declare error as string or undefined? or partial?  (so we don't duplicate `results`)

Reply:
unfortunately no 😔 (I couldn't make it work nicely, it refused calls with results but no error)